website/blog/2022/05/19/weekly-report-0519.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Biweekly Report｜Explore the weeks of May 1st - May 15th
+keywords:
+- Apache APISIX
+- API Gateway
+- Weekly Report
+- Contributor
+description: The Apache APISIX Community Weekly Report hopes to help community members better understand the weekly progress of the Apache APISIX community and and facilitate everyone to participate in the Apache APISIX community.
+tags: [Events]
+---
+
+> From May 1st to May 15th, 35 contributors submitted 77 commits for Apache APISIX. Thank you all for your contributions to Apache APISIX. It is your selfless contribution to make the Apache APISIX project better!
+
+<!--truncate-->
+
+## Introduction
+
+Apache APISIX has grown as a community from the first day of open source and has quickly become the most active open source API gateway project in the world. These achievements cannot be achieved without the joint efforts of our community partners.
+
+"If you want to go fast, go alone.If you want to go far, go together." The Apache APISIX Community Weekly Report hopes to help community members better understand the weekly progress of the Apache APISIX community and and facilitate everyone to participate in the Apache APISIX community.
+
+We have also compiled some issues suitable for newcomers to the community to participate in! If you are interested, don't miss it!
+
+## Contributor Statistics
+
+![Contributors List](https://static.apiseven.com/202108/1652941223501-a03655b5-122f-4fa5-9406-3f3b33093002.png)
+
+![New Contributors](https://static.apiseven.com/202108/1652941259760-bc336da9-7659-4b1e-ac89-d4073bd24c5d.png)
+
+## Good first issue
+
+### Issue #7052
+
+**链接**: https://github.com/apache/apisix/issues/7052
+
+**问题描述**：As a User, I want to use oAuth2 with [PKCE](https://oauth.net/2/pkce/) support, so that I can configure an oAuth2 connection without using client/secret.
```

Comment:
```suggestion
**Description**: As a User, I want to use oAuth2 with [PKCE](https://oauth.net/2/pkce/) support, so that I can configure an oAuth2 connection without using client/secret.
```

Reply:
done